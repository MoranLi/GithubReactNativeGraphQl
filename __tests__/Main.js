import renderer from 'react-test-renderer';
import 'react-native';
import React from 'react';
import MainScreen from '../components/Main.jsx';

it('renders correctly', () => {
  const tree = renderer.create(<MainScreen />);
  expect(tree).toMatchSnapshot();
});
