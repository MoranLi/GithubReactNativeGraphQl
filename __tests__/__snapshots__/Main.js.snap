// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "paddingTop": 50,
      "textAlign": "center",
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://avatars.githubusercontent.com/u/16531358?v=4",
      }
    }
    style={
      Object {
        "display": "flex",
        "height": 58,
        "margin": "auto",
        "width": 66,
      }
    }
  />
  <Text>
    morganli
  </Text>
  <Text>
    MoranLi
  </Text>
  <Text>
    this is bio
  </Text>
  <Text>
    https://www.linkedin.com/in/yukun-li/
  </Text>
  <Text>
    moran.li@usask.ca
  </Text>
  <Text>
    2016-01-04T01:24:51Z
  </Text>
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Repos: 79
      </Text>
    </View>
  </View>
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Following: 11
      </Text>
    </View>
  </View>
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Follower: 8
      </Text>
    </View>
  </View>
</View>
`;
